snowpack/src/build/import-resolver.ts

```diff
@@ -119,3 +120,37 @@ export function createImportResolver({fileLoc, config}: {fileLoc: string; config
     return false;
   };
 }
+
+/**
+ * Create a import glob resolver function, which converts any import globs relative to the given file at "fileLoc"
+ * to a local file. These will additionally be transformed by the regular import resolver, so they do not need
+ * to be finalized just yet
+ */
+export function createImportGlobResolver({fileLoc, config}: {fileLoc: string; config: SnowpackConfig}) {
+  return async function importGlobResolver(spec: string): Promise<string[]> {
+    if (spec.startsWith('/')) {
+      spec = path.posix.join(config.root, spec);
```

Comment:
This is the first hurdle for Windows.
- `if (spec.startsWith('/')) {`: this is fine—since the spec itself is following a URL format, we can count on `/` being a valid value
- `spec = path.posix.join(config.root)`: this is where I think it starts to get tripped up—what if `config.root` is `C:\Users\user\code`? That would produce `C:\Users\user\code**/*.js` Now we’ve overwritten our original value with something that’s neither in Windows format or POSIX/url format

My suggestion would be to preserve the original code `spec` as-written, which will 100% be forward-slash. But as you resolve it, let it be either `/` on *nix, or `\\` on Windows. It’s really tough to properly resolve local filepaths in Windows trying to force them to be forward slashes; I’d recommend just letting it be in `\` until the very end when you need to write it back to code (and you can use the `slash` package to do so)